lib/ssl/src/ssl_certificate.erl

```diff
@@ -295,13 +347,33 @@ do_certificate_chain(CertDbHandle, CertsDbRef, Chain, SerialNr, Issuer, _, ListD
 	    {ok, undefined, lists:reverse(Chain)}
     end.
 
+find_alternative_root([Cert | _], CertDbHandle, CertDbRef, InvalidatedList) ->
+    OtpCert = public_key:pkix_decode_cert(Cert, otp),
+    case find_issuer(#cert{der=Cert, otp=OtpCert}, CertDbHandle, CertDbRef, [], InvalidatedList) of
+        {error, issuer_not_found} ->
+            unknown_ca;
+        {ok, {SerialNr, IssuerId}} ->
+                case ssl_manager:lookup_trusted_cert(CertDbHandle, CertDbRef, SerialNr, IssuerId) of
```

Comment:
Too much indentation here?

Reply:
Strange Emacs agrees and that is what I use, will fix!